html-forms/index.html

```diff
@@ -6,29 +6,44 @@
 <body>
   <h1>Coding Form!</h1>
   <form action="">
-    <div>Email
-      <input type="text" name="email"/>
+    <div class="email">
+      <label for="email">Email</label>
+    </div>
+    <input type="text" name="email" id="email"/>
+
+    <div class="why is coding cool">
+      <label for="why is coding cool">Why is coding cool?</label>
     </div>
-    <div>Why is coding cool?</div>
     <select name="select an option">
-      <option value="select an option">Select an option</option>
-      <option value="becasue it's fun">Because it's fun</option>
-      <option value="because it's interesting">Becasue it's interesting</option>
-      <option value="becasue it's powerful">Becasue it's powerful</option>
+       <option value="select an option">Select an option</option>
+       <option value="becasue it's fun">Because it's fun</option>
+       <option value="because it's interesting">Becasue it's interesting</option>
+       <option value="becasue it's powerful">Becasue it's powerful</option>
     </select>
-    <div>Favorite Language (choose one)
-      <br/>
+
+    <div class="favorite language">
+      <label for="favorite language">Favorite language (choose one)</label>
```

Comment:
This doesn't need to be a label, it is not labelling a specific form control element.